### PR TITLE
Add GA information in README.md

### PR DESCRIPTION
**Pull request by swcloud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,15 @@ Idiomatic Ruby client for [Google Cloud Platform](https://cloud.google.com/) ser
 * [API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs)
 * [google-cloud on RubyGems](https://rubygems.org/gems/google-cloud)
 
+This client supports the following Google Cloud Platform services at a [General Availability (GA)](#versioning) quality level:
+
+* [Cloud Datastore](#cloud-datastore-beta) (GA)
+* [Stackdriver Logging](#stackdriver-logging-beta) (GA)
+* [Cloud Storage](#cloud-storage-beta) (GA)
```
Comment (blowmage) [inline README.md]: Storage isn't going GA, is it? Isn't it still blocked?

Comment (swcloud) [inline README.md]: Let's wait until end of day. :)

Comment (blowmage) [inline README.md]: Agreed.

**Comment by blowmage** [comment]
These changes have been made to the README, except for Storage, which is still Beta. Closing this PR as it is now out of date.

**Pull request closed without merging**